latex/Research_Strategy.tex

```diff
@@ -595,31 +595,90 @@
 
 %Aim 4 - Coordinate, run, and analyze blind challenges to advance modeling of binding (1.5 pages)
 {\bf Aim 4. Coordinate, run, and analyze blind challenges to advance modeling of binding.}
-The data collected in Aims 1-3 will drive annual SAMPL blind challenges, allowing the field to test the latest methods and force fields to assess progress, compare them against one another head-to-head, and perform sensitivity analysis to learn how much different factors (protonation state, tautomer selection, solvent model, force field, sampling method, etc.) affect predictive power. Results will then feed back into improved treatment of these factors for subsequent challenges, driving regular cycles of application, learning, and advancement.
+While the data generated in Aims 1-3 is valuable in its own right, the real impact of this work comes from using this data to conduct regular SAMPL blind challenges which test the state of the art, drive new methodological and force field innovations, allow comparative evaluation of methods, and result in downstream improvements.
+The new, progressive nature of the data generated for SAMPL6-10 means that these challenges will build on one another, and for success in later SAMPL challenges, participants must build on lessons learned from prior challenges.
+SAMPL challenges will thus serve to guide improvements to quantitative modeling of binding, facilitating regular cycles of application, learning, and improvement of models.
+
+% I didn't have this in the outline (bizarrely) but we need to briefly explain how we actually run a SAMPL challenge:
+SAMPL challenges will have submission deadlines once yearly, though not every data type or component of Aims 1-3 will be annual.
```

Comment:
How do we counter the criticism that this takes too much time for participating groups? Perhaps we can offer tools to help reduce human time by facilitating automation of both predictions and analysis?